x-pack/plugins/secops/package.json

```diff
@@ -9,9 +9,10 @@
     "@types/boom": "^7.2.0",
     "@types/color": "^3.0.0",
     "@types/lodash": "^4.14.110",
-    "@types/react-beautiful-dnd": "^7.1.2"
+    "@types/react-beautiful-dnd": "^10.0.1"
   },
   "dependencies": {
+    "react-beautiful-dnd": "^10.0.0",
```

Comment:
I would keep both of these the same number as in `^10.0.1` here even though yarn should update the latest `10.0.x` version I would still keep them the same.

Reply:
You need to do `yarn kbn bootstrap' and the error above will go bye bye